package.json

```diff
@@ -3,22 +3,33 @@
   "description": "Internet website for the ReactJS Dallas User Group",
   "version": "1.0.0",
   "author": "Mike Mathew <m2mathew@me.com>",
+  "engines": {
+    "node": "^8.15.1",
+    "npm": "6.x"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/reactjs-dallas/reactjs-dallas-site"
+  },
   "dependencies": {
+    "@babel/polyfill": "^7.2.5",
     "@material-ui/core": "^3.6.1",
+    "@material-ui/icons": "^3.0.1",
     "dotenv": "^6.1.0",
     "firebase": "^5.5.3",
     "gatsby": "^2.0.64",
     "gatsby-plugin-glamor": "^2.0.7",
-    "gatsby-plugin-manifest": "^2.0.11",
-    "gatsby-plugin-offline": "^2.0.19",
     "gatsby-plugin-react-helmet": "^3.0.4",
+    "gatsby-plugin-remove-serviceworker": "^1.0.0",
     "gatsby-source-filesystem": "^2.0.11",
     "gatsby-transformer-remark": "^2.1.15",
     "glamor": "^2.20.40",
+    "jss": "latest",
```

Comment:
bold move my friend ;)

Reply:
Yeah, maybe not so wise, eh?

Reply:
I am locking this back to `^9.8.7`, the latest version numbered.